src/main/java/frc/robot/RobotContainer.java

```diff
@@ -10,34 +10,40 @@
 import frc.robot.subsystems.DriveTrain;
 import frc.robot.commands.ExampleCommand;
 import frc.robot.subsystems.ExampleSubsystem;
+import frc.robot.subsystems.Limelight;
+import frc.robot.subsystems.Shooter;
+import frc.robot.subsystems.Turret;
 import edu.wpi.first.wpilibj2.command.Command;
 import edu.wpi.first.wpilibj.Joystick;
 import edu.wpi.first.wpilibj2.command.button.JoystickButton;
 import frc.robot.CONSTANTS.*;
 
 /**
- * This class is where the bulk of the robot should be declared.  Since Command-based is a
- * "declarative" paradigm, very little robot logic should actually be handled in the {@link Robot}
- * periodic methods (other than the scheduler calls).  Instead, the structure of the robot
- * (including subsystems, commands, and button mappings) should be declared here.
+ * This class is where the bulk of the robot should be declared. Since
+ * Command-based is a "declarative" paradigm, very little robot logic should
+ * actually be handled in the {@link Robot} periodic methods (other than the
+ * scheduler calls). Instead, the structure of the robot (including subsystems,
+ * commands, and button mappings) should be declared here.
  */
 public class RobotContainer {
 
-  //Controllers
+  // Controllers
   private final Joystick movementJoystick = new Joystick(CONTROLS.JOYSTICK_PORT);
-	private final Joystick arcadeJoystick = new Joystick(CONTROLS.ARCADE_JOYSTICK_PORT);
+  private final Joystick arcadeJoystick = new Joystick(CONTROLS.ARCADE_JOYSTICK_PORT);
 
-  //Subsystems
+  // Subsystems
   private final DriveTrain driveTrain = new DriveTrain();
-  //Commands
-  private final ArcadeDrive arcadeDrive = new ArcadeDrive(driveTrain,
-			() -> movementJoystick.getY(),
-			() -> movementJoystick.getX(),
-			() -> movementJoystick.getRawButtonPressed(CONTROLS.JOYSTICK.TRIGGER),
-			() -> movementJoystick.getRawButtonReleased(CONTROLS.JOYSTICK.TRIGGER));
+  private final Turret turret = new Turret();
+  private final Shooter shooter = new Shooter();
+  public final static Limelight limelight = new Limelight();
```

Comment:
This shouldn't be public nor static.

Reply:
I kept this public so I could access this in the shooter for the found() and onTarget() method

Reply:
nvm